core/backend/src/rpc-impl/RpcBriefcaseUtility.ts

```diff
@@ -58,8 +58,13 @@ export class RpcBriefcaseUtility {
           const fileName = resolver({ briefcaseId, iModelId });
           if (IModelJsFs.existsSync(fileName)) {
             const briefcaseDb = BriefcaseDb.findByFilename(fileName);
-            if (briefcaseDb !== undefined)
-              return briefcaseDb as BriefcaseDb;
+            if (briefcaseDb !== undefined) {
+              if (briefcaseDb instanceof BriefcaseDb) {
```

Comment:
do you need the if on L61? would 62 be a stricter check that suffices?

Reply:
Not really, because line 65 needs to run only if briefcaseDb is not undefined. I guess I could change else to an else if not undefined.

Reply:
I prefer the current flow, but we should probably still avoid `instanceof` where possible...
```suggestion
              if (briefcaseDb.isBriefcaseDb()) {
```

Reply:
This PR would be a good time to finally get rid of "RpcBriefcase" entirely. If you're using Rpc, you can't have a Briefcase. I think it only comes up for the now-deprecated `SyncMode.PullOnly` and `SyncMode.PullAndPush` use cases that are only used by tests, and should be eliminated.